CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 2.22.0-rc.0 / 2020-10-07
+
+* [CHANGE] web: Remove APIv2. #7935
+* [FEATURE] React UI: Implement missing TSDB head stats section. #7876
```

Comment:
This seems more an enhancement than a whole feature.

Reply:
Changed to enhancement.